### DISCUSSION TITLE
Hе работает функция AutoPost :-(

### DISCUSSION THREAD

**Discussion by nykov27** [maxsite/cms#479] [General]
Здравствуйте.
У меня что-то не работает функция AutoPost: размещаю текстовый файл на жесткий диск, но при попытке его загрузить – он горит красным, см. скриншот. На сайт разумеется он не загружается. 
С самим текстовым файлом все в полнейшем порядке, он полностью соответствует синтаксису AutoPost. 
Прошу подсказать, в чем может крыться проблема.
Спасибо. 
![scrin1](https://user-images.githubusercontent.com/97519528/148933833-7249c612-a8b3-45f7-a765-c2e84d42a8ed.png)

**Comment by maxsite** [comment]
Проверьте появляется ли каталог uploads/_temp/. Может на uploads нужно поставить права 777.

**Reply by nykov27** [reply]
Спасибо, посмотрю, чуть позже отпишусь.

**Reply by nykov27** [reply]
Большое спасибо за помощь, проблема была именно в этом!
